android/app/src/main/AndroidManifest.xml

```diff
@@ -6,7 +6,7 @@
          additional functionality it is fine to subclass or reimplement
          FlutterApplication and put your custom class here. -->
     <application
-        android:name="io.flutter.app.FlutterApplication"
+        android:name=".Application"
```

Comment:
The Flutter version is already greater than 1.12, so no need to integrate FCM in such a way, evert back to io.Flutter.app.FlutterApplcation